app/src/ui/release-notes/release-notes-dialog.tsx

```diff
@@ -82,11 +82,47 @@ export class ReleaseNotes extends React.Component<IReleaseNotesProps, {}> {
     )
   }
 
+  /**
+   * If there is just one release, it returns it. If multiple, it merges the release notes.
+   */
+  private getDisplayRelease = () => {
+    if (this.props.newReleases.length === 1) {
+      return this.props.newReleases[0]
+    }
+
+    const earliestRelease = this.props.newReleases.slice(-1)[0]
+
+    let enhancements = new Array<ReleaseNote>()
+    let bugfixes = new Array<ReleaseNote>()
+
+    for (const r of this.props.newReleases) {
+      enhancements = enhancements.concat(r.enhancements)
+      bugfixes = bugfixes.concat(r.bugfixes)
+    }
```

Comment:
Given how few releases we'll be iterating over here I think it's worth mapping to get rid of the `let`s and `for`

```suggestion
    const enhancements = this.props.newReleases.flatMap(x => x.enhancements)
    const bugfixes = this.props.newReleases.flatMap(x => x.bugfixes)
```

Reply:
I forgot flatMap existed!